_sass/_main.scss

```diff
@@ -535,24 +535,6 @@ body {
     }
   } // </section.our-work>
 
-  section.blog {
```

Comment:
I would leave this here, so that if OCTO ever wants to blog about something, they can. I can imagine (in the medium future) that someone writes a post about the work that's being done, or a call to recruit folks.